### DISCUSSION TITLE
I Need  stream URL with the webrtc:// prefix

### DISCUSSION THREAD

**Discussion by SuzukazeAoba13** [bluenviron/mediamtx#4082] [Questions]
### Question

Hello! I have already converted the RTSP stream to WebRTC, and I can play the WebRTC stream via an HTTP URL. However, I need to obtain the original stream URL with the webrtc:// prefix. How can I do this?

**Comment by SuzukazeAoba13** [comment]
Since I don't need the built-in player, I want to integrate the WebRTC stream into my own player. Therefore, I need the stream with the webrtc:// prefix.

**Comment by aler9** [comment]
Hello, i'm not aware of the existence of a `webrtc://` scheme and i do not know any player that supports such scheme.
Currently, WebRTC streams can be ingested into existing players by using the WHEP protocol, that is described in the [README](https://github.com/bluenviron/mediamtx?tab=readme-ov-file#webrtc) and uses the `http://` scheme.

which player are you interested into?

**Reply by SuzukazeAoba13** [reply]
You're very welcome, I understand now. It was my mistake in terms of the concept earlier.
